modules/custom/d_p_carousel/templates/field--field-d-p-cs-item-reference.html.twig

```diff
@@ -53,28 +53,19 @@
 %}
 
 {% if label_hidden %}
-  {% if multiple %}
```

Comment:
Ta sama uwaga co pierwsza - tu powinniśmy mieć możliwość dodania wielu carousel items, bo to nie ma związku z samym d_p_image. Czyli pewnie powinna zostać tylko zmiana do formularza w hook_form_alter.